### PR TITLE
Feature: Email confirmation

### PR DESCRIPTION
**Pull request by alaca**
## Description

This PR adds Email tags `{matching_company}` and `{matching_company_guidelines}`

## Affects

Email template

## Testing Instructions

Add `{matching_company}` and `{matching_company_guidelines}` to the email template.
Make a donation
Check email

## Pre-review Checklist

-   [x] Relevant `@unreleased` tags included in DocBlocks
-   [x] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

### CODE REVIEW

File: src/DoubleTheDonation/FormExtension/Template/Variations/Default.tsx [hunk 1]
```diff
@@ -44,33 +43,17 @@ export default ({inputProps: {name}, label}) => {
             if (response.ok) {
                 const companies = await response.json();
 
-                const filteredCompanies = companies.filter((company: CompanyApi) => {
```
Comment (jonwaldstein) [inline src/DoubleTheDonation/FormExtension/Template/Variations/Default.tsx:47]: Was removing this intentional?

File: src/DoubleTheDonation/FormExtension/Email/EmailTags.php [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+                'context' => 'dtd',
+            ],
+            [
+                'tag'     => 'matching_company_guidelines',
+                'desc'    => __(
+                    'Matching Company Guidelines',
+                    'give-double-the-donation'
+                ),
+                'func'    => [$this, 'handleTag'],
+                'context' => 'dtd',
```
Comment (jonwaldstein) [inline src/DoubleTheDonation/FormExtension/Email/EmailTags.php]: Looks like we never had email tags in v2 forms huh?

I'm wondering if we should prefix the tags and descriptions so the add-on is more clear like  `{double_the_donation_matching_company}`
 _Double the Donation Matching Company_

Do you think that's necessary?

Comment (alaca) [reply src/DoubleTheDonation/FormExtension/Email/EmailTags.php]: We could, but I don't think it's necessary. It is already somewhat prefixed with `matching_`. We could update the tag description instead.

File: src/DoubleTheDonation/FormExtension/Template/Variations/Default.tsx (hunk 1 above)
Comment (alaca) [reply src/DoubleTheDonation/FormExtension/Template/Variations/Default.tsx:47]: Yep.  I spoke to Grace from Double The Donation and she said we shouldn't filter companies.

Comment (jonwaldstein) [reply src/DoubleTheDonation/FormExtension/Template/Variations/Default.tsx:47]: Who is _Grace from Double The Donation_? haha! thanks for the clarification

File: src/DoubleTheDonation/FormExtension/Email/EmailTags.php (hunk 2 above)
Comment (jonwaldstein) [reply src/DoubleTheDonation/FormExtension/Email/EmailTags.php]: Okay, let's go ahead and update the descriptions then to help clarify for customers 👍

Comment (alaca) [reply src/DoubleTheDonation/FormExtension/Email/EmailTags.php]: Resolved https://github.com/impress-org/give-double-the-donation/pull/20/commits/3f180c29e4e224c8242bb94b689fa570e949ed41

**Review by jonwaldstein — APPROVED** [review]
Great job!

**Pull request merged**